jwst/assign_wcs/miri.py

```diff
@@ -310,5 +310,10 @@ def alpha_beta2XanYan(input_model, reference_files):
                       'mir_coroncal': imaging,
                       'mir_lrs-fixedslit': lrs,
                       'mir_lrs-slitless': lrs,
-                      'mir_mrs': ifu
+                      'mir_mrs': ifu,
+                      'mir_flatmrs': not_implemented_mode,
+                      'mir_flatimage': not_implemented_mode,
+                      'mir_flat-mrs': not_implemented_mode,
+                      'mir_flat-image': not_implemented_mode,
```

Comment:
mir_flat-mrs and mir_flat-image are obsolete (superseded by mir_flatmrs and mir_flatimage), but I guess it doesn't hurt to leave them in there for backwards compatibility. Otherwise this list is good and complete. And the assignments look good too - there's no reason to have WCS coverage for flats and darks.